sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -211,7 +211,10 @@ private void processNonCachedEvent(final @NotNull SentryBaseEvent event) {
   private void setThreads(final @NotNull SentryEvent event) {
     if (event.getThreads() != null) {
       for (SentryThread thread : event.getThreads()) {
-        thread.setCurrent(MainThreadChecker.isMainThread(thread));
+        Boolean isCurrentThread = thread.isCurrent();
```

Comment:
```suggestion
        final Boolean isCurrentThread = thread.isCurrent();
```

Reply:
Should I rather inline the call, now that it's just the null check?